bazelrunner/src/main/kotlin/org/jetbrains/bsp/bazel/bazelrunner/BazelInfo.kt

```diff
@@ -16,12 +16,16 @@ data class BazelRelease(
   val major: Int
 ) {
 
-  fun mainRepositoryReferencePrefix(isBzlModEnabled: Boolean) = when (major) {
+  fun isRelativeWorkspacePath(label: String) = when (major) {
     in 0..3 -> throw RuntimeException("Unsupported Bazel version, use Bazel 4 or newer")
-    in 4..5 -> "//"
-    else ->
-      if (isBzlModEnabled) "@@//"
-      else "@//"
+    in 4..5 -> label.startsWith("//")
+    else -> label.startsWith("@//") || label.startsWith("@@//")
+  }
+
+  fun stripPrefix(label: String) = when (major) {
+    in 0..3 -> throw RuntimeException("Unsupported Bazel version, use Bazel 4 or newer")
+    in 4..5 -> label.substring("//".length)
```

Comment:
i think it's better to use `removePrefix` here

Reply:
Fixed! I couldn't find the method before.